webapps-schema/src/main/java/io/camunda/webapps/schema/entities/operation/BatchOperationEntity.java

```diff
@@ -108,7 +108,8 @@ public BatchOperationEntity setSortValues(final Object[] sortValues) {
   }
 
   public void generateId() {
-    setId(UUID.randomUUID().toString());
+    final long operationReference = UUID.randomUUID().getMostSignificantBits();
```

Comment:
Why do we change also `BatchOperationEntity` id? It's not necessary for our purpose, or?

Reply:
Just to give some more context here: `batchOperationId` is smth visible to the users in UI and also smth that will be stored in `list-view` index for the purpose of filtering process instances by this value. It feels safe, but I would prefer to keep changes smaller not to touch broader functionality. What do you think?

Reply:
I can revert this change. I just thought they were very similar and it felt natural to change both to be alike. but will leave this alone